Utils/src/BTagScale.cc

```diff
@@ -0,0 +1,392 @@
+// -*- C++ -*-
```

Comment:
The wildly inconsistent indentation in this file makes it difficult to read. Please revise if you have the time.